### DISCUSSION TITLE
How does the multiplayer works?

### DISCUSSION THREAD

**Discussion by Tupiet** [OpenTTD/OpenTTD#9858] [Q&A]
Hey! I'm a bit new at this game and this is my first time asking here, so I don't know if I'm in the correct place.

My question is about the multiplayer mode. I want to play with my friends to OpenTTD in multiplayer mode. I though I needed to open a specific port in my router if I was the server, but then I read [this blog post](https://www.openttd.org/news/2021/09/26/new-multiplayer-experience) and I got really confused: do the server still needs to have the port opened, or it's not necessary anymore?

Thanks!

**Comment by glx22** [comment]
Opening the ports is now optional, but it's still better to open them if possible.

**Reply by Tupiet** [reply]
Thanks for answering that fast. If that's the case, can I ask how would I do it in order to play it without opening ports? I mean, what should I have to do? I tried it and couldn't find the way. Thanks!

**Reply by glx22** [reply]
You just start a server, in `public` or `invite-only` mode, and the magic happens.

**Reply by Tupiet** [reply]
Thanks! I just had to wait a few minutes in order to start the server. Really thanks!
